server/plugin/virtual_agent.go

```diff
@@ -317,16 +317,15 @@ func (p *Plugin) getPostActionOptions(options []Option) []*model.PostActionOptio
 	return postOptions
 }
 
-func (p *Plugin) createMessageAttchment(fileID string) (*MessageAttachment, string) {
+func (p *Plugin) createMessageAttachment(fileID string) (*MessageAttachment, error) {
 	var attachment *MessageAttachment
 	fileInfo, appErr := p.API.GetFileInfo(fileID)
 	if appErr != nil {
-		return nil, fmt.Sprintf("Error getting file info. Error: %s", appErr.Message)
+		return nil, fmt.Errorf("error getting file info. Error: %s", appErr.Error())
```

Comment:
```suggestion
		return nil, fmt.Errorf("error getting file info. Error: %w", appErr)
```
We use `%w` for errors. Change this wherever applicable.